package.json

```diff
@@ -19,9 +19,13 @@
   "homepage": "https://github.com/codeforanchorage/plug-at-20-sms#readme",
   "dependencies": {
     "body-parser": "^1.15.0",
+    "chai": "^3.5.0",
     "cron": "^1.1.0",
     "express": "^4.13.4",
+    "fs": "0.0.1-security",
```

Comment:
the fs module is included in node, no need to add it to package.json. i'm surprised this works actually.